### ISSUE TITLE
Improve "Send test notification" button in Issue alerts

### ISSUE THREAD

**Issue by rodolfoBee** [getsentry/sentry#77765]
### Problem Statement

When a user click the "Send test notification" sentry will send a test event to the project. This test event does not trigger the alert in all cased leading to bad user experience.

### Solution Brainstorm

Change the logic of the button to actually perform the actions configured in the alert rule, not try to send a test event to the project.

### Product Area

Alerts

**Comment by ceorourke** [comment]
It works for me. Can you add details about the rule's configuration? Maybe the action doesn't work correctly?

**Comment by rodolfoBee** [comment]
The alert rule is to create a new Jira issue and it was not triggered by the new issue. I made a new test here:
https://dev-curumas.sentry.io/alerts/rules/springboot/15461879/details/

**Comment by ceorourke** [comment]
Can they manually create a Jira ticket through an issue successfully (in the same project, with the same settings they've configured for the alert)?

**Comment by rodolfoBee** [comment]
Yes, i can manually create Jira issue with the same settings used in the alert.

**Comment by schew2381** [comment]
Hi thanks for the bug report, I'll take a look at this

**Comment by dcheng1248** [comment]
Just to add that I ran into the same problem just now. Tried to send a test notification thinking it would email some users, instead an issue was created that was not relevant to the conditions set in the alert. It led to some confusion until I saw this issue.

**Comment by kcons** [comment]
There's work underway to revamp that page and notifications in general, and this is something we're aiming to fix there.
In the meantime, I'll see if we can clarify what that button does.

**Comment by cobyeastwood183** [comment]
https://sentry.zendesk.com/agent/tickets/163813
